### ISSUE TITLE
subcommands missing

### ISSUE THREAD

**Issue by TheMeier** [xorpaul/g10k#5]
in order to be a r10k replacement all subcommands should be implemented.

COMMANDS
    deploy         Puppet dynamic environment deployment
    help           show help
    puppetfile     Perform operations on a Puppetfile
    version        Print the version of r10k

**Comment by alexharv074** [comment]
At the very least, there should be syntactic equivalence.

Most basic usage:

```
r10k puppetfile install 
```

becomes

```
g10k -puppetfile
```

**Comment by maxadamo** [comment]
Dear Andreas, 
I want to connect a question to this ticket (to avoid polluting your repo with too many issues). 
Do you think it is beyond the scope of the tool to create switch names, which are  compatible with r10k? 
Why am I asking this? 
It's not a functionality that I personally need, but following up a question that I have read in the puppetcommunity slack channel, some people is wondering if g10k can be used by Code Manager (from puppet Enterprise). 
I don't even know if that would be the only problem. I also don't know if it's more convenient to create a wrapper rather than adding unnecessary complexity to the code.

**Comment by alexharv074** [comment]
Yeah lack of syntactic equivalence is a shame because if we had it then anyone could just insert a line in their code somewhere:

```
alias r10k=/usr/local/bin/g10k
```

I am sure the small user base of g10k would understand if a breaking change was made to migrate to use r10k's switch names.

**Comment by xorpaul** [comment]
I'll have to think about it.

To be honest the only reason the g10k parameters look the way they do is because of the `flag` go library.
